Classes/TMDebugConsole.m

```diff
@@ -186,27 +185,42 @@ - (void)setUpDateFormatter
 
 - (void)logMessage:(DDLogMessage *)logMessage
 {
+	logMessage = [self formatLogMessage:logMessage];
+	[self scheduleLogMessage:logMessage];
+}
+
+- (DDLogMessage *)formatLogMessage:(DDLogMessage *)logMessage
+{
+	if (formatter)
+	{
+		NSString *logMsg = [formatter formatLogMessage:logMessage];
+		if (logMsg != nil)
+		{
+			logMessage->logMsg = logMsg;
+		}
+		else
+		{
+			logMessage = nil;
+		}
+	}
+	return logMessage;
```

Comment:
You are right I'll simplify the code a little bit in another pull request, since this one is already merged